ush/eva/jedi_gsi_compare_conv.yaml

```diff
@@ -349,7 +349,7 @@ diagnostics:
             x:
               variable: experiment::ErrDiff::${variable}
             y:
-              variable: experiment::MetaData::air_pressure
+              variable: experiment::MetaData::pressure
```

Comment:
@PraveenKumar-NOAA this will need to be changed in your version of this file until it's merged in

Reply:
@CoryMartin-NOAA thanks, it worked.